lib/moon/design/form/addon/btn.ex

```diff
@@ -0,0 +1,20 @@
+defmodule Moon.Design.Form.Addon.Btn do
```

Comment:
1. Why it is Addon.Btn? Would Form.SubmitActionButton or something similar make more sense?
2. Did this come from Design system and is agreed or it is some arbitary change?

Reply:
2. it's not from the design system, but I have seen 2 simillar problems - eye on a password and save inside textrare. they are both fixable in terms of headless ui by the same approach
1. It's not about a submit or any action at all. it's just a shorthand to put button "visually inside" textarea as i've seen on a screen.

Reply:
2. the same way is suitable for adding some counters "18/255 symbols" and so on

Reply:
2. had a short discussion with @dkireev - he agrredd to leave this "add-on system" as an experiment

Reply:
Ok, but please do not use Srt Fn Nms, Bcs Thy r hrd t rd.